scripts/test_env.py

```diff
@@ -127,12 +129,20 @@ def run(self, is_genesis: bool = False):
         os.environ["TORII_P2P_ADDR"] = f"{self.host_ip}:{self.p2p_port}"
         os.environ["TORII_API_URL"] = f"{self.host_ip}:{self.api_port}"
 
-        genesis_arg = "--submit-genesis" if is_genesis else ""
+        if submit_genesis:
+            os.environ["IROHA_GENESIS_PRIVATE_KEY"] = self.private_key
+            # Assuming it was copied to the peer's directory
+            os.environ["IROHA_GENESIS_FILE"] = str(self.peer_dir.joinpath("genesis.json"))
+        else:
+            for x in ["IROHA_GENESIS_PRIVATE_KEY", "IROHA_GENESIS_FILE"]:
+                os.environ.pop(x, None)
```

Comment:
https://docs.python.org/3/library/subprocess.html#subprocess.Popen accepts an `env` parameter, allowing to specify the process environment variables without relying on global state. 

The caveat is that it will require copying the "base" `os.environ`, as it doesn't have any inheritance logic.

Reply:
Thanks for the catch. I think it would be more clear to opt-out from `os.environ` fully and always specify `env` explicitly.

Reply:
It's not always safe to opt-out from the `os.environ`, there are variables like `SystemRoot` on windows and `PATH` on linux.

I _think_ iroha will be fine, but it might lead to unnecessary and potentially hard-to-debug problems imo..

Reply:
Thanks for the catch again. Will refactor in a way so that initial `os.environ` data is also used.

Reply:
Refactored.